tests/MongoDB.Driver.Tests/MongoDB.Driver.Tests.csproj

```diff
@@ -2,14 +2,25 @@
 
   <PropertyGroup>
     <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <LangVersion>6</LangVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningsAsErrors />
     <IsPackable>false</IsPackable>
     <CodeAnalysisRuleSet>..\..\MongoDBTest.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <AssemblyTitle>MongoDB.Driver.Tests</AssemblyTitle>
+    <Product>MongoDB.Driver.Tests</Product>
+    <Company>MongoDB Inc.</Company>
+    <Copyright>Copyright © 2010-present MongoDB Inc.</Copyright>
```

Comment:
This is really minor, but if we're using the copyright symbol, we could potentially also use the en-dash "–" instead of a hyphen the range of dates. I don't mind one way or another though!

Reply:
Acknowledged!